src/main/java/io/openliberty/tools/intellij/util/LibertyGradleUtil.java

```diff
@@ -167,7 +167,11 @@ private static boolean containerVersion(String plugin) {
      */
     public static String getGradleSettingsCmd(Project project) throws LibertyException {
         GradleProjectSettings gradleProjectSettings = GradleSettings.getInstance(project).getLinkedProjectSettings(project.getBasePath());
-        if (gradleProjectSettings.getDistributionType().isWrapped()) {
+        if (gradleProjectSettings == null) {
+            String translatedMessage = LocalizedResourceUtil.getMessage("gradle.settings.is.null");
+            throw new LibertyException("Cannot execute project because there is an error with configured Gradle. Make sure to configure a valid path for Gradle inside IntelliJ Gradle preferences.", translatedMessage);
```

Comment:
```suggestion
            throw new LibertyException("Could not execute action because there is an error with Gradle configuration. Make sure to configure a valid path for Gradle inside IntelliJ Gradle preferences.", translatedMessage);
```